inc/taxonomy-content-language.php

```diff
@@ -9,7 +9,7 @@
  */
 function wmf_register_content_language_taxonomy(): void {
 	$language_type_args = array(
-		'heirarchical' => false,
+		'hierarchical' => false,
```

Comment:
Unrelated, but the typo bugged me :)

This doesn't change any behavior, because the default for 'hierarchical' is already false.